docs/en/sdk-manual/http-api.md

```diff
@@ -0,0 +1,215 @@
+| app_version      | NO       | String    | 1.1.0                                                                                                                                                                                                                                                                                                                                            | Application version number                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| app_title        | NO       | String    | shopping                                                                                                                                                                                                                                                                                                                                         | Application name                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| items            | NO       | Object    | [{<br/>	&nbsp;&nbsp;"id": "b011ddc3-632f-47cb-a68a-ad83678ecfed",<br/>	&nbsp;&nbsp;"name": "Classic coat-rack",<br/>	&nbsp;&nbsp;"category": "housewares",<br/>	&nbsp;&nbsp;"price": 167<br/>}]                                                                                                                                                  | Item list, Supports uploading multiple items at one time. A maximum of 100 items can be uploaded at one time<br>For the item quantity limit, please refer to [Event and Attribute Limitation](./web.md#event-and-attribute-limitation) <br>For the supported attributes of item, please refer to: [item attribute](./web.md#item-attributes)                                                                                                          |
```

Comment:
-> For the supported attributes of the item, please refer to [item attribute](./web.md#item-attributes)